<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="3" passed="2" failed="1" skipped="0">
  <reporter-output>
  </reporter-output>
  <suite started-at="2023-02-17T02:26:13 IST" name="Default suite" finished-at="2023-02-17T02:27:26 IST" duration-ms="72946">
    <groups>
    </groups>
    <test started-at="2023-02-17T02:26:13 IST" name="Default test" finished-at="2023-02-17T02:27:26 IST" duration-ms="72946">
      <class name="PHPTravels.Tests.test_checkLogin">
        <test-method is-config="true" signature="intializeTest()[pri:0, instance:PHPTravels.Tests.test_checkLogin@7ec7ffd3]" started-at="2023-02-17T02:26:13 IST" name="intializeTest" finished-at="2023-02-17T02:26:22 IST" duration-ms="9026" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- intializeTest -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:PHPTravels.Tests.test_checkLogin@7ec7ffd3]" started-at="2023-02-17T02:26:22 IST" name="checkLogin" data-provider="getData" finished-at="2023-02-17T02:26:52 IST" duration-ms="29247" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[user@phptravels.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[demouser]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:PHPTravels.Tests.test_checkLogin@7ec7ffd3]" started-at="2023-02-17T02:26:52 IST" name="checkLogin" data-provider="getData" finished-at="2023-02-17T02:26:57 IST" duration-ms="5332" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[abcd@phptravels.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[abcd]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertTrue(Assert.java:45)
at org.testng.Assert.assertTrue(Assert.java:55)
at PHPTravels.Tests.test_checkLogin.checkLogin(test_checkLogin.java:58)
at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base/java.lang.reflect.Method.invoke(Method.java:578)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method signature="checkLogin(java.lang.String,java.lang.String)[pri:0, instance:PHPTravels.Tests.test_checkLogin@7ec7ffd3]" started-at="2023-02-17T02:26:57 IST" name="checkLogin" data-provider="getData" finished-at="2023-02-17T02:27:26 IST" duration-ms="28939" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[user@phptravels.com]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[demouser]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkLogin -->
        <test-method is-config="true" signature="endTest()[pri:0, instance:PHPTravels.Tests.test_checkLogin@7ec7ffd3]" started-at="2023-02-17T02:27:26 IST" name="endTest" finished-at="2023-02-17T02:27:26 IST" duration-ms="313" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endTest -->
      </class> <!-- PHPTravels.Tests.test_checkLogin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
